packages/zowe-explorer/src/tools/ZoweTerminal.ts

```diff
@@ -186,7 +186,9 @@ export class ZoweTerminal implements vscode.Pseudoterminal {
 
     private async handleEnter() {
         this.write(ZoweTerminal.Keys.NEW_LINE);
-        const cmd = this.command;
+        const isAsyncCommand = this.command.startsWith(":async");
+        const isForgetCommand = this.command.startsWith(":forget");
+        const cmd = this.command.substring(isAsyncCommand ? 6 : isForgetCommand ? 7 : 0).trim();
```

Comment:
Could we avoid using nested ternary here? It's definitely cleaner and more concise to use ternary, but I think we can do the operation without nesting another check 😋 

```suggestion
        const cmd = this.command.substring(isAsyncCommand || isForgetCommand ? 6 + Number(isForgetCommand) : 0).trim();
```

Reply:
Thanks for the suggestion! 🤯